dartagnan/src/main/java/com/dat3m/dartagnan/expression/utils/Utils.java

```diff
@@ -21,7 +21,21 @@ public static BooleanFormula generalEqual(Formula f1, Formula f2, SolverContext
 		// Fallback
 		return fmgr.getIntegerFormulaManager().equal(convertToIntegerFormula(f1, ctx), convertToIntegerFormula(f2, ctx));
 	}
-	
+
+	public static BooleanFormula generalZero(Formula f, SolverContext ctx) {
```

Comment:
`generalEqualZero` is probably a better name

Reply:
I agree.